connectors/sources/confluence.py

```diff
@@ -206,6 +214,12 @@ async def _handle_client_errors(self, url, exception):
 
             await self._sleeps.sleep(retry_seconds)
             raise ThrottledError
+        elif exception.status == 401:
```

Comment:
nit, can we add handling `400` bad reqest here for completness?  every now and then we do see client request errors for some edge cases so it would be good to log some more details here